packages/shared/src/components/post/PostEngagements.tsx

```diff
@@ -61,6 +62,9 @@ function PostEngagements({
 
   const onCommented = (comment: Comment, isNew?: boolean) => {
     if (isNew) {
+      postAnalyticsEvent(AnalyticsEvent.CommentPost, post, {
+        extra: { commentId: comment.id, origin: 'comment modal' },
```

Comment:
I see you import origin, why don't you use the origin enum here 🙏

Reply:
Oh, right. Missed that one. Will utilize the enum before merging.